examples/mockups/index.html

```diff
@@ -0,0 +1,329 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="/dist/css/site.css" />
+    <script src="/dist/js/vendor.js"></script>
+    <script src="/dist/js/site.js"></script>
+  </head>
+  <body>
+
+    <!-- Top Menu 
+    ----------------------------------- -->
+    <div class="ui menu topnav fixed borderless">
```

Comment:
`borderless` is a native SUI class. It removes the borders from the menu items: https://semantic-ui.com/collections/menu.html.

As far as I can understand form SUI docs, `attached` is used to attach a menu to another component while `fixed` is used to fix the menu to the window, using `position: fixed`.

The only custom additions I did here were:
* Adding the class `topnav`.
   * This will allow us to address the topnav for multiple purposes. Even future javascripts. I think we need to keep this but can maybe turn into `id` instead of `class`.
   * Instead of adding multiple class names and making multiple .overrides to those classes, I added this style:
   https://github.com/readthedocs/site-community/pull/1/files#diff-cc4e63c51189136d3f2bafb692036f86bac2f88c6ee1219fb3f6ed7cdf4d8692R21-R24
   We talked about this already after your review and I'm changing this to use only the native SUI here. Should I keep the `topnav` class/id?

* The other custom CSS I did here was to style the menu items:
https://github.com/readthedocs/site-community/pull/1/files#diff-cc4e63c51189136d3f2bafb692036f86bac2f88c6ee1219fb3f6ed7cdf4d8692R33-R55
I would also argue in favor of using custom CSS here, specific for the `topnav` because we will want the top menu to have a custom look different from any other menu we might want on the site and/or application. Creating some customization here leaves the framework menus free to be reused anywhere else without mimicking the top menu.

Reply:
> borderless is a native SUI class. It removes the borders from the menu items

:+1: that one is new to me!

> As far as I can understand from SUI docs, attached is used to attach a menu to another component 

Yup, also used with segments that connect, like the top navigation and the section immediate below.

> I also added the following to avoid multiple class names and making some .overrides to those classes

If I were to work on this, I would look for these overrides in `menu.overrides` and get a little confused that they aren't there. Splitting these rules out to a separate component structure does seem like a premature optimization and the structure will be difficult to work with.

The rules attached to `topnav` seem minor enough, considering you don't want that menu to look like either menus.

However I would probably say let's get back to a pretty basic include structure and remove the need for the `templates/` includes. There seems to be some additional complexity here and I don't think we're going to have such as complex structure that we benefit from this pattern. SUI overrides are already hard to digest, so there is wisdom in keeping things easy to work on and remember.

Reply:
> There seems to be some additional complexity here and I don't think we're going to have such as complex structure that we benefit from this pattern.
👍 Ok. Makes sense.